iotdb-core/node-commons/src/assembly/resources/conf/iotdb-system.properties.template

```diff
@@ -0,0 +1,1874 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+####################
+### Cluster Configuration
+####################
+
+# Used for indicate cluster name and distinguish different cluster.
+# effectiveMode: first_start
+# Datatype: string
+cluster_name=defaultCluster
+
+####################
+### Node RPC Configuration
+####################
+
+# Used for RPC communication inside cluster.
+# Could set 127.0.0.1(for local test) or ipv4 address.
+# effectiveMode: first_start
+# Datatype: String
+cn_internal_address=127.0.0.1
```

Comment:
we can comment all parameter now for consistency

Reply:
fix

Reply:
fix